server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1424,7 +1424,7 @@ private void getMasterLock(final String zMasterLoc) throws KeeperException, Inte
 
       sleepUninterruptibly(TIME_TO_WAIT_BETWEEN_LOCK_CHECKS, TimeUnit.MILLISECONDS);
     }
-
+    log.info("Got Master lock.");
```

Comment:
Could make the wording consistent across all the services. I would use "Attempting to acquire" and "Acquired". You can also drop the periods off the end of the log message. They aren't complete sentences anyway.